packages/core/src/JWEditor.ts

```diff
@@ -195,12 +195,16 @@ export class JWEditor {
             if (!shortcut.platform && this._platform === Platform.MAC) {
                 shortcut.pattern = shortcut.pattern.replace(/ctrl/gi, 'CMD');
             }
-            this.eventManager.keymap.addShortuct(
-                shortcut.pattern,
-                shortcut.commandId,
-                shortcut.commandArgs,
-                priority,
-            );
+            if (shortcut.commandId) {
+                this.eventManager.keymap.addShortuct(
+                    shortcut.pattern,
+                    shortcut.commandId,
+                    shortcut.commandArgs,
+                    priority,
+                );
+            } else {
+                this.eventManager.keymap.removeShortuct(shortcut.pattern, priority);
+            }
```

Comment:
I'm really not sure `removeShortcut` adds value. I mean, to remove a shortcut, people *have* to *create* a new shortcut that *does nothing* anyway, they *have* to do it that way, so why pretend in the code ?